odl/tomo/geometry/conebeam.py

```diff
@@ -19,67 +19,54 @@
 
 # Imports for common Python 2/3 codebase
 from __future__ import print_function, division, absolute_import
-from abc import ABCMeta
 from future import standard_library
 standard_library.install_aliases()
+from abc import abstractmethod
 from builtins import super
 
 # External
 import numpy as np
 
 # Internal
-from odl.util.utility import with_metaclass
 from odl.set.domain import IntervalProd
 from odl.discr.grid import TensorGrid
 from odl.tomo.geometry.detector import Flat2dDetector
 from odl.tomo.geometry.geometry import Geometry
 
 
-__all__ = ('ConeBeamGeometry', 'ConeFlatGeometry', 'CircularConeFlatGeometry',
+__all__ = ('ConeBeamGeometry', 'CircularConeFlatGeometry',
            'HelicalConeFlatGeometry',)
 
 
-class ConeBeamGeometry(with_metaclass(ABCMeta, Geometry)):
+class ConeBeamGeometry(Geometry):
```

Comment:
Thats kindoff why I wanted the rename to `DivergentBeamGeometry` since this does not have a strong 3d connotation.